src/lib/deviceSpecs/specifications.js

```diff
@@ -479,6 +514,15 @@ const specs = {
 			address: '0x00030000',
 			alt: '0'
 		},
+		otaRegion: {
+			address: '0x80289000',
+			alt: '2'
+		},
+		otaFlag: {
```

Comment:
This might seem like a nit-pick, but repeated data like this is possibly best factored out into an instance that is reused.